src/views/Accessibility/AccessibiltyRequest/Documents/index.tsx

```diff
@@ -50,12 +51,17 @@ const AccessibilityDocumentsList = ({
         Cell: ({ row }: any) => (
           <>
             <UswdsLink
+              aria-describedby={`doc-link-${row.original.id}`}
+              target="_blank"
               asCustom={Link}
-              to={`/some-508-request/${row.original.name}`}
+              to={row.original.url}
             >
               {t('documentTable.view')}
+              <span className="usa-sr-only">document type</span>
             </UswdsLink>
-            <span className="usa-sr-only">{row.original.name}</span>
+            <span className="usa-sr-only" id={`doc-link-${row.original.id}`}>
```

Comment:
I _think_ Sai mentioned something about having this text inside the href, and not outside it, but I am unsure on my memory of those events. Might be worth checking in.

Reply:
I will check with him. I was doing some research and read that you have to be careful with having different text within the link than is shown or read. This can cause issues for folks using voice-recognition to control their computers.

But we're already kind of doing that, so I'll make sure we chat and update the implementation as needed.

Reply:
[WCAG guidance](https://www.w3.org/TR/WCAG20-TECHS/G201.html) is suggesting we include 'opens in a new tab' as a part of the `<a> `tag.  @jim I'd love to know more about the issue you're referring to so we can make sure we don't cause it accidentally.